### PR TITLE
log allow list, metadata rules and converted json schema

### PR DESCRIPTION
**Pull request by OsmondJiang**
easy for us to debug metadata validation issues in docfx v3

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5664)

### CODE REVIEW

File: src/docfx/config/ops/OpsMetadataRuleConverter.cs [hunk 1]
```diff
@@ -13,6 +13,9 @@ internal static class OpsMetadataRuleConverter
     {
         public static string GenerateJsonSchema(string rulesContent, string allowlistsContent)
         {
+            Log.Write(rulesContent);
+            Log.Write(allowlistsContent);
```
Comment (yufeih) [inline src/docfx/config/ops/OpsMetadataRuleConverter.cs]: @928PJY , will this cause vscode extension to crash?

Comment (OsmondJiang) [reply src/docfx/config/ops/OpsMetadataRuleConverter.cs]: no, vscode would not output Verbose logs

Comment (928PJY) [reply src/docfx/config/ops/OpsMetadataRuleConverter.cs]: @yufeih @OsmondJiang  For now, `--verbose` is not default enabled, but I am going to add a vscode config to make the user be able to enable that, but before that I will figure out a way to fix the crash caused by stdout size limitation, by make the limitation larger or flush the buffer when it catch some threshold.
anyway, for now, it is OK to add this log

**Comment by OsmondJiang** [comment]
> Wait for integration test result to see the impact on log and performance

Seems OK

**Comment by yufeih** [comment]
> > Wait for integration test result to see the impact on log and performance
> 
> Seems OK

I don't see any logs here https://dev.azure.com/ceapex/Engineering/_build/results?buildId=131583&view=logs&jobId=c4527c29-b924-5b47-b3c6-092a857c3f6e&j=c4527c29-b924-5b47-b3c6-092a857c3f6e&t=03eedd41-1a06-5244-c198-e8d85c0de60b

**Comment by OsmondJiang** [comment]
> > > Wait for integration test result to see the impact on log and performance
> > 
> > 
> > Seems OK
> 
> I don't see any logs here https://dev.azure.com/ceapex/Engineering/_build/results?buildId=131583&view=logs&jobId=c4527c29-b924-5b47-b3c6-092a857c3f6e&j=c4527c29-b924-5b47-b3c6-092a857c3f6e&t=03eedd41-1a06-5244-c198-e8d85c0de60b

Here: 
https://github.com/dotnet/docfx/pull/5664#issuecomment-603133167

**Pull request merged**